### DISCUSSION TITLE
Seek operation failed

### DISCUSSION THREAD

**Discussion by HubKing** [ebandal/H2Orestart#28] [Q&A] [answered]
아래와 같은 오류가 발생하면서 문서가 안 열리는 경우가 꽤 있던데, 왜 그런 것인가요? 예를 들면 다음 사이트의 첨부 파일에 있는 HWP 파일 두 개 모두 그렇습니다: https://blog.naver.com/nsesibong/60024483421

![image](https://github.com/ebandal/H2Orestart/assets/10900489/98d558eb-f50b-45b2-8a7e-7ee33bcba288)

**Comment by ebandal** [comment, marked as answer]
해당 사이트에 게시된 HWP파일 두개 모두 한글 3.0 포맷의 문서입니다.   이전 포맷으로 작성된 문서입니다.
텍스트편집기로 열어 보면  "HWP Document File V3.00" 으로 시작합니다.
![image](https://github.com/ebandal/H2Orestart/assets/114871013/8307e5c9-28c5-46e1-a6cd-4a3114de17a1)
한글 프로그램에서 "한글문서(97~3.0)" 포맷으로 저장되거나,  오래전 버전의 한글 프로그램에서 작성된것으로 보입니다.

H2Orestart 확장은  한글 5.0 포맷과  HWPX 포맷을 지원하며,  3.0 포맷은 지원하지 않습니다.
다시 말하면,  한글 3.0 포맷은  H2Orestart확장을 거치지 않고,  리브레오피스에서 직접 열립니다.  
예로 들어주신 오류창은  리브레오피스에서  3.0 포맷을 처리하던 중 발생한 오류인것 같습니다.
리브레오피스에서 오류가 발생하는 이유는 저는 알지 못합니다.

**Reply by HubKing** [reply]
오늘 AUR에서 업데이트하니까 위의 파일들이 열리네요. 그림이 다 깨어져 보이는 것 같기는 한데, 어쨌든 열리니 텍스트는 읽을 수 있습니다.

![image](https://github.com/ebandal/H2Orestart/assets/10900489/38c3cc62-45ab-499d-bcc0-ab06b65b8abf)
